src/mixins/behaviors.js

```diff
@@ -87,12 +86,11 @@ export default {
     _.invoke(this._behaviors, 'unbindUIElements');
   },
 
-  _triggerEventOnBehaviors: function(args) {
-    var triggerMethod = _triggerMethod;
+  _triggerEventOnBehaviors: function(...args) {
     var behaviors = this._behaviors;
     // Use good ol' for as this is a very hot function
     for (var i = 0, length = behaviors && behaviors.length; i < length; i++) {
-      triggerMethod(behaviors[i], args);
+      triggerMethod.call(behaviors[i], ...args);
```

Comment:
This seems very, very weird. Should this not be `triggerMethod.apply(behaviours[i], args)`